server/protocols/jmap-rfc-8621/src/test/scala/org/apache/james/jmap/json/MailboxSerializationTest.scala

```diff
@@ -127,9 +127,9 @@ class MailboxSerializationTest extends AnyWordSpec with Matchers {
           |      "Storage":{"used":19}
           |    }
           |  }
-          |}""".stripMargin)
+          |}""".stripMargin
 
-      assertThatJson(new Serializer(new TestId.Factory).serialize(MAILBOX)).isEqualTo(expectedJson)
+      assertThatJson(Json.prettyPrint(new Serializer(new TestId.Factory).serialize(MAILBOX))).isEqualTo(expectedJson)
```

Comment:
Why do we need Json.prettyPrint in an assertion?

Reply:
To have a string out of that JsValue. You would prefer to use `stringify` instead, or extract this piece of code?

Reply:
stringify is better IMO